### DISCUSSION TITLE
module 'pybullet' has no attribute 'addUserDebugPoints'

### DISCUSSION THREAD

**Discussion by manavkulshrestha** [bulletphysics/bullet3#4417] [General]
`p.addUserDebugPoints([p.getBasePositionAndOrientation(o1)[0]], [[0,1,0]])`

Maybe I'm doing something very stupid, but why isn't this method not available? I see it in the docs. I'm using pybullet 3.2.5

`Requirement already satisfied: pybullet==3.2.5 in /home/mk/anaconda3/envs/rp/lib/python3.10/site-packages (3.2.5)`

**Comment by ManuCorrea** [comment]
I am just wondering but this might be the case, do you have a pip pybullet installation outside conda? Either pip or pip3 installation could mess if you have the same package in conda.

**Comment by adelapen** [comment]
In case someone else runs into this, I was able to work around this issue by creating a new Conda environment and doing `pip3 install PyBullet` instead of using the [conda-forge PyBullet package](https://anaconda.org/conda-forge/pybullet).
